src/Uno.UI.BindingHelper.Android/Uno.UI.BindingHelper.Android.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras" ToolsVersion="15.0">
 	<PropertyGroup>
-		<TargetFrameworks>MonoAndroid12.0;MonoAndroid13.0;net461;xamarinios10;netstandard2.0;xamarinmac20</TargetFrameworks>
```

Comment:
This is an Android-specific project. It doesn't make sense to have xamarinios10 or xamarinmac20 at all. I think the binaries produced by these builds are never used.